R/object_name_linter.R

```diff
@@ -45,7 +45,12 @@ object_name_xpath <- local({
 #'
 #' @param styles A subset of
 #'   \Sexpr[stage=render, results=rd]{lintr:::regexes_rd}. A name should
-#'   match at least one of these styles.
+#'   match at least one of these styles. The `"symbols"` style refers to
+#'   names containing *only* non-alphanumeric characters; e.g., `%+%` from
+#'   ggplot2 or `%>%` from magrittr will not generate lint markers, whereas
```

Comment:
I think "would" is probably grammatically more appropriate, as in "`%>%` from magrittr _would_ not generate lint markers". This helps convey that usage of other packages' objects won't throw lints regardless of style, only objects "you" define. That is, _using_ `%m+%` won't throw a lint, but _defining_ it will.